GDC_mapping.yml

```diff
@@ -19,17 +19,26 @@ ResearchSubject:
 Diagnosis:
     Mapping:
         id: 'cases.diagnoses.diagnosis_id'
+        #identifier:
```

Comment:
Are we intending to use these in the future?

Reply:
I believe we are - I simply want to uncomment and update the hand-written transformation files on another release since it is related to a separate JIRA ticket